### DISCUSSION TITLE
Mass Update of Attribute Values

### DISCUSSION THREAD

**Discussion by paultvis** [atrocore/atrocore#501] [Q&A]
Hi All,

Is is possible to do mass updates of product attribute values? It seems that if you select products, then the mass update option only shows properties of the products, or lets you add relationships 

I have checked through the documentation but cannot see anything about this. With mass update, this must surely be the most common activity to do so I cannot believe it can't be done.

Thanks

Paul

**Comment by atrolex** [comment]
Via user interface currently no, via API already yes.

**Reply by paultvis** [reply]
Is that planned? We really like the flexibility that Atro gives us and it lets us do a lot more than we could do in Akeneo. However the fact that only some values (the standard properties) and not all values that a product contains can be updated en mass really impacts the workflow and makes the task much more prone to errors. Something that is an everyday task for companies that import product catalogues from suppliers (setting values on attributes let us add value to the information we provide) will now take hours whereas in akeneo, with the ability to mass update attributes means it took minutes. Within families many of the attribute values are common across many products and the ability for users to quickly set common attributes across product groups within a family means that the final pass only requires a small number of attributes to be set individually for each product.  As you have the ability to do this through the API, does it mean that adding the feature is on the backlog?

Thanks

Paul

**Reply by atrolex** [reply]
It is on the roadmap, but with a low priority. We wait for someone who will be ready to partly finance it. We need 8 hours. Price is 480 EUR fix. It will be implemented same way like for the fields – a single value can be set to multiple chosen products. If attribute is marked as unique, it cannot be mass updated.
